tests/tests/test-contract-loops.ts

```diff
@@ -18,6 +18,7 @@ describeWithMoonbeam("Moonbeam RPC (Contract Loops)", `simple-specs.json`, (cont
   it("should increment contract state - to check normal contract behavior", async function () {
     // // instantiate contract
     const contract = await deployContractManualSeal(
+      context.polkadotApi,
```

Comment:
You could pass the context directly instead of each library

Reply:
>I thought it would look cleaner 

it would look increasingly better as I move all api related calls to this file